sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedForEpkRange.ts

```diff
@@ -423,7 +443,15 @@ export class ChangeFeedForEpkRange<T> implements ChangeFeedPullModelIterator<T>
         getEmptyCosmosDiagnostics(),
       );
     } catch (err) {
-      // If any errors are encountered, eg. partition split or gone, handle it based on error code and not break the flow.
+      if (err.code >= 400 && err.code !== StatusCodes.Gone) {
```

Comment:
Do we have a **StatusCodes.BadRequest** that we can use?

Reply:
updated